bin/create.js

```diff
@@ -156,8 +169,19 @@ module.exports = Cli.createCommand('create', 'Create webtasks', {
                 if (argv.containerLimit) parseHash(argv, 'containerLimit');
 
                 if (argv.watch && argv.json) {
-                    throw new Error('The `watch` flag can not be enabled at the same time '
-                        + 'as the `json` flag.');
+                    throw Cli.usageError('The --watch flag can not be enabled at the same time '
+                        + 'as the --json flag.');
+                }
+                
+                if (argv.cluster_url || argv.token) {
+                    if (argv.profile !== tokenOptions.profile.default) {
+                        throw Cli.usageError('The --profile option cannot be '
+                            + 'combined with either --cluster_url or --token.');
```

Comment:
Can we simplify this by saying any of (url, token, container) can be specified on the command line, and the rest is taken from profile?